tests/improved-sanity-test/main.yml

```diff
@@ -58,6 +58,10 @@
       tags:
         - redhat_subscription
 
+    - role: docker_pull_run_remove
+      tags:
+        - docker_pull_run_remove
+
```

Comment:
Maybe we can  remove this role pre-upgrade and run one time after the upgrade? I think there are other tests in the suite that validate images and containers go through the upgrade successfully (i.e. cockpit).  This role validates that there are no issues pulling and running known images which we are probably only concerned about in the HEAD.